### PR TITLE
Change int to decimal to support more sensors

### PR DESCRIPTION
**Pull request by oskary**
Using a decimal instead of int should make it compatible with non Ikea sensors too, for example the Aqare Temperature and umidity Sensor T1

### CODE REVIEW

**Comment by hagronnestad** [comment]
Thank you! I will double check that this works with my IKEA sensor and merge after.

**Comment by hagronnestad** [comment]
I haven't noticed that the IKEA sensor doesn't even support decimal points, that's kind of bad.

I think using `double` instead of `decimal` would be more appropriate though, would you mind updating the PR?

File: Dirigera.Lib/Dirigera.Lib/Models/EnvironmentSensor.cs [hunk 1]
```diff
@@ -6,7 +6,7 @@ namespace Dirigera.Models
 {
     public class EnvironmentSensor : Device
     {
-        public int CurrentTemperature { get; internal set; }
+        public decimal CurrentTemperature { get; internal set; }
```
Comment (hagronnestad) [inline Dirigera.Lib/Dirigera.Lib/Models/EnvironmentSensor.cs]: I think `double` is a more appropriate type to use.

File: Dirigera.Lib/Dirigera.Lib/Models/EnvironmentSensor.cs [hunk 2]
```diff
@@ -25,7 +25,7 @@ internal override void PopulateFromDto(DeviceDto dto)
             // Extract the "standard" attributes from the Attributes dictionary into their own properties
             if (dto.Attributes is not null)
             {
-                if (dto.Attributes.ContainsKey("currentTemperature")) CurrentTemperature = dto.Attributes["currentTemperature"].GetInt32();
+                if (dto.Attributes.ContainsKey("currentTemperature")) CurrentTemperature = dto.Attributes["currentTemperature"].GetDecimal();
```
Comment (hagronnestad) [inline Dirigera.Lib/Dirigera.Lib/Models/EnvironmentSensor.cs]: See above comment and use `GetDouble()` instead,

Comment (oskary) [reply Dirigera.Lib/Dirigera.Lib/Models/EnvironmentSensor.cs]: Changed!

**Review by hagronnestad — APPROVED** [review]

**Comment by hagronnestad** [comment]
Thank you, a new version of the Nuget package has been released: https://www.nuget.org/packages/Dirigera.Lib/1.0.6

Here's the GitHub release: https://github.com/hagronnestad/dirigera/releases/tag/1.0.6

**Pull request merged**